### PR TITLE
#150889069 JSEND bookmark responses

### PR DESCRIPTION
**Pull request by andela-oowonikoko**
#### What does this PR do?

JSEND bookmark responses

#### Description of Task to be completed?

- JSEND bookmark responses and update the tests
- Fix a bug of identifying valid `resource_id` when bookmarking a resource

#### How should this be manually tested?

At the root of the project directory run the following command:
```$pytest -v```

#### What are the relevant pivotal tracker stories?

#150889069

### CODE REVIEW

File: api/views/bookmarks.py [hunk 1]
```diff
@@ -65,8 +74,7 @@ def post(self, resourceType, resource_id):
                     "createdAt": bookmark.created_at
                 }
             },
-            "status": "success",
-            "message": "{} was bookmarked successfully".format(resource_type)
+            "status": "success"
```
Comment (LarryWachira) [inline api/views/bookmarks.py]: Still think there should be a message here. Perhaps nest it inside `data` like mai did. But that's just a suggestion and it's fine either way.

**Comment by LarryWachira** [comment]
LGTM

File: api/views/bookmarks.py (hunk 1 above)
Comment (andela-oowonikoko) [reply api/views/bookmarks.py]: Alright, will fix it

**Comment by danielwangai** [comment]
@andela-oowonikoko I reworked the entire Bookmarks story. I think that since due to the changes made, we should wait for it (`PR 96`) to be merged first so as to effect this changes to it.

What do you think?

**Comment by andela-oowonikoko** [comment]
@danielwangai That's fine by me

**Comment by Andela-ddiei** [comment]
LGTM.

**Pull request merged**